### PR TITLE
Notice for qubes-tunnel testing

### PR DESCRIPTION
**Pull request by tasket**
Issues #3503 #3520

### CODE REVIEW

**Comment by andrewdavidwong** [comment]
I'm *slightly* worried about the wording here:
> and is recommended for most users on Qubes 3.2 and 4.0

@marmarek, any opinion?

**Comment by tasket** [comment]
I chose that wording because I feel that attempts to use the new scripts (in service already under different naming) will lead to fewer issues for users than the existing method... despite the testing status.

But please feel free to re-word it as you see fit.

**Comment by tasket** [comment]
Also wondering, since qubes-tunnel appears to be more of a direct contribution (per Patrick's indication in #3503 ) instead of a 'Qubes-contrib' optional package, whether it should just be added to the Qubes testing repositories.

File: configuration/vpn.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ redirect_from:
 How To make a VPN Gateway in Qubes
 ==================================
 
+## Note on new qubes-tunnel availability!
+
+A streamlined VPN option is in testing phase and is recommended for most users on Qubes 3.2 and 4.0. It is currently [available here](https://github.com/tasket/qubes-doc) and qubes-users discussion is [here](https://groups.google.com/d/msgid/qubes-users/ee24f104-efbc-23f7-aca3-6be86104ddaf%40posteo.net).
```
Comment (andrewdavidwong) [inline configuration/vpn.md]: Is `https://github.com/tasket/qubes-doc` really the correct URL?

**Comment by andrewdavidwong** [comment]
> I chose that wording because I feel that attempts to use the new scripts (in service already under different naming) will lead to fewer issues for users than the existing method... despite the testing status.
> 
> But please feel free to re-word it as you see fit.

I guess the reason for my uncertainty is that I don't know what our official position regarding that package is. It seems somewhat strange to recommend something from a non-official source to *all* users. I'll leave it for @marmarek to decide.

> Also wondering, since qubes-tunnel appears to be more of a direct contribution (per Patrick's indication in #3503 ) instead of a 'Qubes-contrib' optional package, whether it should just be added to the Qubes testing repositories.

Also for @marmarek to decide. If it's a contrib package, though, we should make sure to follow our procedure for such packages:
https://www.qubes-os.org/doc/package-contributions/

File: configuration/vpn.md (hunk 1 above)
Comment (tasket) [reply configuration/vpn.md]: oops... that should be 'tasket/qubes-tunnel'

**Comment by tasket** [comment]
For reference, Marek and Patrick comments have it as a direct Qubes inclusion:
https://github.com/QubesOS/qubes-issues/issues/3503#issuecomment-361114105

I was just wondering if qubes-testing was a better repository in this case. Then it wouldn't seem as strange to recommend it (or to use wording asking users to help test).

**Comment by andrewdavidwong** [comment]
Closing this for now due to the prolonged lack of activity. If you believe this is a mistake, or if anyone would like this PR to remain open, please leave a comment, and we'll be happy to reopen this. Thank you.

**Pull request closed without merging**